video/out/vulkan/common.h

```diff
@@ -25,6 +25,10 @@
 
 #include <vulkan/vulkan.h>
 
+#if HAVE_WIN32_DESKTOP
+#include <vulkan/vulkan_win32.h>
```

Comment:
Actually, this shouldn't be needed. `vulkan_win32.h` is included by `vulkan.h` when `VK_USE_PLATFORM_WIN32_KHR` is defined, which we define in this file already.

In fact, I think it may be an error to assume `vulkan_win32.h` exists? Not that it matters since there's only one vulkan loader for the time being anyway (and I don't see this changing soon).